common/styles/app.css

```diff
@@ -956,33 +956,28 @@ footer .container1 {
   -ms-user-select: none;
   -webkit-user-select: none;
   user-select: none;
-  background: transparent;
+  background: #fff;
+  border: solid 1px;
+  border-top: none;
```

Comment:
add `border-bottom: none` too. I think because of the calculation that we're doing the main-container might be 1-2px off the actual size and the border-bottom is visible.